### ISSUE TITLE
segfault when exe name is empty in opt-file

### ISSUE THREAD

**Issue by gkarlos** [namhyung/uftrace#944]
As far as i am aware, it is allowed to pass the exe file name in the opt file. When doing so a segfault occurs when the exe name is empty / empty space.

Steps to reproduce:
```
$ echo " " > opt
$ ./uftrace record --opt-file opt
Segmentation fault (core dumped)
```

A segfault also occurs when a file is given but the opt-file is empty. i.e:
```
$ echo -n "" > opt
$ ./uftrace record --opt-file opt t-abc
Segmentation fault (core dumped)
```
Though i think the latter should probably be a separate issue. Let me know and i will update.

**Comment by namhyung** [comment]
Hmm.. I thought it was fixed by your change, but it only for quotes.  OK then I think this is same problem in the parsing the option file, but it'd better to compare the backtraces.  Handling command line argument is a different problem of course.

**Comment by gkarlos** [comment]
@namhyung Yes i had a fix for that too (i think the first case only), but you commented that the PR was solving two issues, invalid file and missing file. Also i think the two cases are not necessarily 100% connected.

**Comment by namhyung** [comment]
Got it.  Can you work on this too?

**Comment by gkarlos** [comment]
Sure. Do you reckon the second case above should be a separate issue?

**Comment by namhyung** [comment]
Yes, you can have the two fixes in a PR.
